docs/UserGuide.md

```diff
@@ -311,25 +315,26 @@ Modifies social media handles that are associated with existing contacts on Soci
 <div markdown="block" class="alert alert-primary">
 **:mag_right: Format:**
 `edit INDEX [fb/FACEBOOK] [ig/INSTAGRAM] [tele/TELEGRAM] [tiktok/TIKTOK] [twitter/TWITTER]`
-* Adds a new handle for contacts who did not have that specific handle pegged with them when they were first added to SociaLite.
-* Replaces the handle for the person at the specified `INDEX` according to the flag and input provided.
-* The index refers to the index number shown on the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
-* The flag refers to any of the following: `fb/` `ig/` `tele/` `tiktok/` `twitter/`
-* User input after the given flag represents the new social media handle associated with the contact.
+* The index must be an **integer ranging from 1 to the last index of the contact list**
+* New handles will be added for contacts who do not have any specific handle pegged with them.
+* If the person at the specified `INDEX` already has a handle for a specific platform flagged in the command, that handle will be replaced.
 * Only social media handles specified as input will be altered. Other social media handles remain unchanged.
+* The flag must be one of the following: `fb/` `ig/` `tele/` `tiktok/` `twitter/`
+* Only one of each flag may be entered per command
```

Comment:
Small nit: add period behind. You might want to check through the entire UG and add in period as well!